docs/breaking_changes.md

```diff
@@ -33,8 +33,7 @@ breaking change. Images pushed by Engine 1.10 to a Registry 2.3 cannot be
 pulled by digest by older Engine versions. A `docker pull` that encounters this
 situation returns the following error:
 
-``` Error response from daemon: unsupported schema version 2 for tag TAGNAME
-```
+``` Error response from daemon: unsupported schema version 2 for tag TAGNAME ```
```

Comment:
Can you change this to;

    ```
    Error response from daemon: unsupported schema version 2 for tag TAGNAME
    ```

That will turn it into a code-block (instead of inline code)